MapboxDirections/MBLaneIndicationComponent.swift

```diff
@@ -0,0 +1,67 @@
+/**
+A component that represents a lane  representation of an instruction.
+ */
+open class LaneIndicationComponent: Component {
```

Comment:
This class needs to be bridgeable to Objective-C, with a class name that begins with the library’s class prefix `MB`.